server/integration_tests/test_data/detection_api_reranking/debug_info.json

```diff
@@ -0,0 +1,80 @@
+{
+  "places_detected": [
+    "california"
+  ],
+  "places_resolved": "(name: California, dcid: geoId/06); ",
+  "main_place_dcid": "geoId/06",
+  "main_place_name": "California",
+  "entities_detected": [],
+  "entities_resolved": "",
+  "query_with_places_removed": "population that is rich in",
+  "sv_matching": {
+    "CosineScore": [
+      0.730574369430542,
+      0.7251521348953247,
+      0.7115399837493896,
+      0.7670478820800781,
+      0.7392960786819458,
+      0.8287845849990845,
+      0.7240197062492371,
+      0.7594512104988098,
+      0.7549042701721191,
+      0.749762773513794,
+      0.8086668252944946,
+      0.722908616065979,
+      0.7306226491928101,
+      0.7177798748016357,
+      0.7594658732414246
+    ],
+    "MultiSV": {
+      "Candidates": [
+        {
+          "AggCosineScore": 0.9053,
+          "DelimBased": false,
+          "Parts": [
+            {
+              "CosineScore": [
+                0.9012579321861267
+              ],
+              "QueryPart": "population",
+              "SV": [
+                "Count_Person"
+              ]
+            },
+            {
+              "CosineScore": [
+                0.9094176888465881
+              ],
+              "QueryPart": "rich",
+              "SV": [
+                "dc/topic/Income"
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    "Query": "population that is rich in",
+    "SV": [
```

Comment:
i don't see post_rerank field in this golden json?

Reply:
Good point, I added it now